ui/src/treeDataProvider/AzureTreeDataProvider.ts

```diff
@@ -43,7 +43,13 @@ export class AzureTreeDataProvider implements TreeDataProvider<IAzureNode>, Disp
         }
 
         this._disposables.push(this._azureAccount.onFiltersChanged(() => this.refresh()));
```

Comment:
This always fires on logged in?

Reply:
Sure looks like it to me. You can follow the trail of events in [this file](https://github.com/Microsoft/vscode-azure-account/blob/master/src/azure-account.ts):

1. `login` calls `updateSessions`
1. `updateSessions` fires `onSessionsChanged`, which triggers `updateSubscriptions`
1. `updateSubscriptions` fires `onSubscriptionsChanged`, which triggers `updateFilters`
1. `updateFilters` triggers `onFiltersChanged`

I tested opening VS Code with '0', '1', 'many', and 'all' filters selected. Also tested logging in/out in the same VS Code session. It was always fired